packages/common/src/adapters/collection.ts

```diff
@@ -190,8 +191,7 @@ export const playlistMetadataForUpdateWithSDK = (
     playlistName: input.playlist_name ?? '',
     description: input.description ?? '',
     coverArtCid: input.cover_art_sizes ?? '',
-    isPrivate: input.is_private ?? false,
-    isImageAutogenerated: input.is_image_autogenerated ?? false
```

Comment:
does this mean we can't remove our custom artwork and get back auto generated ones? not sure if that was desired or not

Reply:
it's a part of create type which is merged here so it's still there